docusaurus/sidebars.js

```diff
@@ -11,7 +11,7 @@ function getFilenamesInDir(prefix, dir, excludes) {
     .readdirSync(dir)
     .filter(
       (fileName) =>
-        !(fileName.endsWith(".inapp.md") || fileName.endsWith("postgres.md") || fileName.endsWith("-migrations.md"))
+        !(fileName.endsWith(".inapp.md") || fileName.endsWith("-migrations.md"))
```

Comment:
This function is used for sources _and_ destinations. We only want to exclude postgres from sources (because it gets manually added to the top later) so we should not filter it out here. Instead we can add it to the `excludes` argument on line 43.